qcodes/instrument_drivers/Lakeshore/Model_325.py

```diff
@@ -472,6 +472,24 @@ def __init__(self, parent: 'Model_325', name: str, loop: int) ->None:
             get_cmd=f"RAMPST? {self._loop}"
         )
 
+        self.add_parameter(
+            "resistance",
+            get_cmd=f"HTRRES? {self._loop}",
+            set_cmd=f"HTRRES {self._loop} {{}}",
+            vals=Enum(25, 50),
```

Comment:
enumeration is not needed once `val_mapping` is used
```suggestion
```

Reply:
All the lines with "vals=Enum(...)" followed by "val_mapping={...}" have been removed. (not only for the two new parameters, but also for a few old ones)